src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -822,10 +822,21 @@ export function areAxisLabelsEnabled(chartOptions: any, axisPropsName: string, s
     };
 }
 
+function shouldExpandYAxis(chartOptions: any) {
+    const min = get(chartOptions, 'xAxisProps.min', '');
+    const max = get(chartOptions, 'xAxisProps.max', '');
+    return min === '' && max === '' ? {} : { getExtremesFromAll: min !== '' || max !== '' };
```

Comment:
please encap `min === '' && max === ''` to a variable and then think about which values would `min !== '' || max !== ''` be in case the encapsulated condition is false